wiz-admission-controller/values.yaml

```
@@ -0,0 +1,92 @@
+# Default values for wiz-admission-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+wizApiToken:
+  clientId: ""
+  clientToken: ""
+  env: "prod" # (prod/ gov)
+
+replicaCount: 2
+
+image:
+  repository: docker.io/library/wiz-admission-controller-server
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose defalt is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+opaWebhook:
+  rules: # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-rules
+    - operations:
+      - CREATE
+      apiGroups:
+      - "*"
+      apiVersions:
+      - "*"
+      resources:
+      - "*"
+  failurePolicy: Ignore # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy
+  sideEffects: None # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#side-effects
+
+  errorEnforcementMethod: AUDIT  # AUDIT/ BLOCK
+  policyEnforcementMethod: BLOCK # AUDIT/ BLOCK 
+  policies: {}
```

What is the type of `policies` field? 
You gave a default value of empty dictionary but in the commented-out examples below it's a list of strings.
```suggestion
  policies: {}
```